package.json

```diff
@@ -18,7 +18,7 @@
     "test": "jest --coverage"
   },
   "dependencies": {
-    "@vivliostyle/core": "^2.1.1",
+    "@vivliostyle/core": "2.1.4",
```

Comment:
`@vivliostyle/core` is still set as `dependencies` because the broker page also uses files not bundled in  CLI such as `vivliostyle-viewport.css`.